### PR TITLE
Change to headless chrome

### PR DESCRIPTION
**Pull request by zlister**
This PR also includes a lot of updates to teaspoon_env.rb for Rubocop

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1804
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: Config files were changed.
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 

Branch | Back End Coverage | Front End Coverage
-- | -- | --
master | [Paste Output Here] | [Paste Screenshot Here]
[New Branch] | [Paste Output Here] | [Paste Screenshot Here]

- [x] Automated regression test(s) pass :N/A

**Reviewer 1:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: spec/teaspoon_env.rb [hunk 1]
```diff
@@ -1,187 +1,251 @@
...
+    # of filenames or regular expressions. The default excludes assets from
+    # vendor, gems and support libraries.
+    coverage.ignore = [%r{spec/}, %r{app/assets/javascripts/templates},
+                       %r{vendor/}, %r{gems/}]
+
+    # Various thresholds requirements can be defined, and those thresholds will
+    # be checked at the end of a run. If any aren't met the run will fail with
+    # a message. Thresholds can be defined as a percentage (0-100), or nil.
+    coverage.statements = 51.3
+    coverage.branches = 34.24
```
Comment (mayerm94) [inline spec/teaspoon_env.rb:247]: Why were the coverage thresholds lowered?

Comment (zlister) [reply spec/teaspoon_env.rb:247]: @losborne made that change. I cherry-picked this file from him to help avoid merge conflicts down the road.

**Review by lukoktonos — APPROVED** [review]
Looks good, teaspoon tests pass locally.

**Review by mayerm94 — APPROVED** [review]
Looks good

**Pull request merged**